src/lot/controller.ts

```diff
@@ -62,6 +69,40 @@ async function scrapLot(_req: express.Request, res: express.Response) {
   );
 }
 
+function addRouteStep(req: express.Request, res: express.Response) {
+  ExpressHandler(
+    async () => {
+      const { lotId, stepId, insertAfterIndex } = req.body;
+
+      const lot = await LotModel.findOne({ _id: lotId }).lean();
```

Comment:
Are these checks for lot and step ids redundant, given the middleware that this endpoint has subscribed to?